src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Represents the user interface of the chat bot.
+ * A <code>Ui</code> object corresponds to the user interface of the chat bot.
+ */
+public class Ui {
+    private Scanner scanner;
+
+    /**
+     * Constructs a <code>Ui</code> object.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Reads the next line of input from the user.
+     * @return The next line of input from the user.
+     */
+    public String readCommand() {
+        return scanner.nextLine();
+    }
```

Comment:
```suggestion
    public String getCommand() {
        return scanner.nextLine();
    }
```
Since this is a getter method, should this be named as `getCommand`?

Reply:
You are absolutely correct, thank you for being an amazing reviewer <3.